### PR TITLE
Add detect_errors command attribute to tag_pileup_frequency

### PR DESCRIPTION
**Pull request by gregvonkuster**
Also apply standard discussed here https://github.com/galaxy-iuc/standards/issues/43

### CODE REVIEW

File: tools/tag_pileup_frequency/tag_pileup_frequency.xml [hunk 1]
```diff
@@ -3,31 +3,29 @@
...
+-b '$input1'
+-i '${input1}.bai'
+-c '$input2'
+-s $tag_shift
+-n $bin_size
+-e $tags_equal
+-r $reads
+-p $proper_pe
+-a $combine_strands
+-t '\${GALAXY_SLOTS:-4}'
```
Comment (nsoranzo) [inline tools/tag_pileup_frequency/tag_pileup_frequency.xml]: You cannot use single-quotes here or the shell won't replace `$GALAXY_SLOTS` with its value, keep the double quotes.

File: tools/tag_pileup_frequency/tag_pileup_frequency.xml [hunk 2]
```diff
@@ -3,31 +3,29 @@
...
+-w $composite_smoothing_window
+-m 'true'
```
Comment (nsoranzo) [inline tools/tag_pileup_frequency/tag_pileup_frequency.xml]: No need to quote.

**Comment by gregvonkuster** [comment]
Done - thanks @nsoranzo ;)

**Comment by nsoranzo** [comment]
Thanks @gregvonkuster!

**Pull request merged**